lib/src/main/kotlin/com/swmansion/starknet/data/types/Transaction.kt

```diff
@@ -739,8 +697,9 @@ sealed class DeployAccountTransaction : Transaction() {
     @SerialName("class_hash")
     abstract val classHash: Felt
 
+    // not in RPC spec
     @SerialName("contract_address")
-    abstract val contractAddress: Felt
+    abstract val contractAddress: Felt?
```

Comment:
```suggestion
    abstract val contractAddress: Felt? = Felt.ZERO,
```

I think this should probably have a default value?

Reply:
It's an abstract field - it cannot have default value at this level, its default value is provided in deploy account v1/v3.

Reply:
Ahh yes, sorry I am dzban